W-13925085-usage-reports-transition (#186)
mulesoft/docs-general: modules/ROOT/pages/pricing.adoc

This document describes the changes introduced in Runtime Manager for CloudHub 1.0, 
CloudHub 2.0, Anypoint Runtime Fabric, and Access Management to support the new pricing and packaging model.

== Usage Reports Changes from vCore to the New Pricing and Packaging Model

> When your organization opts into the new usage pricing and packaging model, your apps must be redeployed. After redeploying of your apps, your usage reports temporarily display both vCore-based usage and the new usage metrics for your apps. Usage reports cover an entire month, so usage reports appear for both the new usage pricing model and the vCore-based usage pricing for that month.

Reviewer: ```suggestion
When your organization opts into the new usage pricing and packaging model, your apps must be redeployed. After redeploying your apps, your usage reports temporarily display both vCore-based usage and the new usage metrics for your apps. Usage reports cover an entire month, so usage reports appear for both the new usage pricing model and the vCore-based usage pricing for that month.
```

> When your organization opts in to the new usage pricing and packaging model, your usage reports temporarily display both vCore-based usage and the new usage metrics for your apps. Your apps must be redeployed after your organization implements the new usage pricing and packaging. Because a usage report covers an entire month, during the month that redeploy happens, usage reports appear for both usage pricing and vCore pricing.

Reviewer: ```suggestion
When your organization opts into the new usage pricing and packaging model, your apps must be redeployed. After redeploying of your apps, your usage reports temporarily display both vCore-based usage and the new usage metrics for your apps. Usage reports cover an entire month, so usage reports appear for both the new usage pricing model and the vCore-based usage pricing for that month.
```
